tests/acceptance/features/webUIRenameFiles/renameFilesWithoutReload.feature

```diff
@@ -0,0 +1,15 @@
+Feature: rename files
+  As a user
+  I want to rename files
+  So that I can organise my data structure
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Alice" has created file "index.jpeg"
+    And user "Alice" has logged in using the webUI
+
+
+  Scenario: Rename file extension with reload
```

Comment:
This scenario can be added to this file https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIRenameFiles/renameFiles.feature , you don't need to make a separate file for that. Maybe you can make this a scenario outline and test different formats too?

Reply:
Ok, I have done it in recent commit.